### PR TITLE
Add Lestnet Chain ID 21363

### PR DESCRIPTION
**Pull request by foxtrotravi**
- Add Lestnet Chain ID 21363

### CODE REVIEW

**Comment by foxtrotravi** [comment]
> Revoked

Can you please tell me why is the prettier action failing and what can I do to fix it?

File: _data/icons/lestnet.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
```
[the remark this answers is not in the document]
Comment (foxtrotravi) [reply _data/icons/lestnet.json:1]: Can you please explain what do you mean by "Revoked access" and how do I fix this?

**Review by ligi — CHANGES_REQUESTED** [review]
I could not resolve your CID with `ipfs get` in a reasonable time  or cloudflare could not resolve the image in a reasonable time for the CI - maybe you have the icon only locally in your ipfs node that is not properly connected or there is some other IPFS problem - and before having a back and forth here I require you to put it in iconDownload. Before this back and forth did cost me a lot of time that I do not have.

ignore comments of @Dqddyvillager1722 - would have blocked the user already - but there is a problem with github in this regard currently

**Comment by foxtrotravi** [comment]
> I could not resolve your CID with `ipfs get` in a reasonable time or cloudflare could not resolve the image in a reasonable time for the CI - maybe you have the icon only locally in your ipfs node that is not properly connected or there is some other IPFS problem - and before having a back and forth here I require you to put it in iconDownload. Before this back and forth did cost me a lot of time that I do not have.
> 
> ignore comments of @Dqddyvillager1722 - would have blocked the user already - but there is a problem with github in this regard currently

Hi @ligi 
I have added the file to iconsDownload folder as you requested and also updated the ipfs url. I have used web3.storage to upload the image as an svg. I even ran the 
`ipfs get bafkreigg4ovuohje6iva42fou7rc7a6fkozvfkg22fhuvhw5oz7vii74ha`
command and it worked for me. 

Can you please review the PR once again? Thanks

**Review by ligi — APPROVED** [review]

**Pull request merged**